dataactvalidator/config/sqlrules/fabs41_detached_award_financial_assistance_6.sql

```diff
@@ -0,0 +1,8 @@
+-- If PrimaryPlaceOfPerformanceCode is XX00000, PrimaryPlaceOfPerformanceZip4 must not be 'city-wide'
+SELECT
+    row_number,
+    place_of_performance_code
```

Comment:
We probably want to also show them `place_of_performance_zip4a` as one of the things we select